docs/modules/operation/pages/telemetryd/protocols/bmp.adoc

```diff
@@ -19,47 +20,49 @@ The BMP parser accepts BMP connections from router packets using a <<telemetryd/
+| Parameter                   | Default    | Description
+3+| *Optional*
+| dnsLookupsEnabled           | true       | Enable or disable DNS resolution of router and peer hostnames.
+| bulkhead.maxConcurrentCalls | 1000       | Limit the number of parallel parsing operations.
+| bulkhead.maxWaitDurationMs  | 5 minutes  | Limit the amount of time to wait for a saturated bulkhead (in milliseconds).
 |===
 
-== Configure BMP Listener on a Minion
+== Configure BMP listener on a minion
 
 To enable and configure a TCP listener for BMP on Minion, connect to the Karaf Console and set the following properties:
 
-[source]
+NOTE: The protocol must also be enabled on {page-component-title} for the messages to be processed.
+
+[source, console]
 ----
 $ ssh -p 8201 admin@localhost
 ...
 admin@minion()> config:edit --alias tcp-5000 --factory org.opennms.features.telemetry.listeners
 admin@minion()> config:property-set name BMP
 admin@minion()> config:property-set class-name org.opennms.netmgt.telemetry.listeners.TcpListener
 admin@minion()> config:property-set parameters.port 5000
-admin@minion()> config:property-set parsers.0.name BMP 
+admin@minion()> config:property-set parsers.0.name BMP
 admin@minion()> config:property-set parsers.0.class-name org.opennms.netmgt.telemetry.protocols.bmp.parser.BmpParser
 admin@minion()> config:update
 ----
 
-NOTE: The protocol must also be enabled on {page-component-title} for the messages to be processed.
-If you do not specify the queue name, the fallback is the name of the parser. 
+If you do not specify the queue name, the fallback is the name of the parser.
 
 [[telemetryd-bmp-statistics-adapter]]
-== BMP Telemetry Adapter
+== BMP telemetry adapter
 
 The BMP telemetry adapter handles BMP statistics received and parsed by the <<telemetryd-bmp-parser, BMP parser>>.
 Statistics received from the router are associated as performance data with that router.
 The router must exist as a regular node in {page-component-title}.
 
-{page-component-title} uses the IP address exporting BMP messages to associate a router with the particular {page-component-title} node.
-In addition, the node's metadata can specify a BGP ID, which will then be used to associate routers.
-If the parameter `metaDataNodeLookup` is not empty, it will be interpreted as a `context:key` metadata name, which will be used to lookup a node that has stored the queried BGP ID as a value in exactly this key.
+{page-component-title} uses the IP address exported in BMP messages to associate a router with the appropriate {page-component-title} node.
+In addition, the node's metadata can be used to specify a BGP ID, which will be used to associate routers.
+If the parameter `metaDataNodeLookup` is not empty, it will be interpreted as a `context:key` metadata name, which will be used to lookup a node that has stored the queried BGP ID as a value in the specified key.
```

Comment:
```suggestion
If the parameter `metaDataNodeLookup` is not empty, it will be interpreted as a `context:key` metadata name, which will be used to look up a node that has stored the queried BGP ID as a value in the specified key.
```